## Introduce

**Build best quality control method on Metagenomic analysis** 

➡️ Statistical proof & ML technic for extracting variables with batch effect and perform covariate adjustment

➡️ Establish the norm for outlier detection 

➡️ Extracting key microbiome that affect each diseases


## Related paper review
1. Feature selection with the Fisher score followed by the Maximal Clique Centrality algorithm can accurately identify the hub genes of hepatocellular carcinoma[ >>](https://www.nature.com/articles/s41598-019-53471-0) 

[00174_Abstract](https://github.com/Jin-Baek/Human-microbiome.research/blob/main/Paper%20review/00174_Abstract.pdf)<br>
[ Keyword ] Maximum Likelihood Estimation, Fisher score


2. Dimensionality reduction for visualizing single-cell data using UMAP[ >>](https://www.nature.com/articles/nbt.4314)

[00181_Abstract](https://github.com/Jin-Baek/Human-microbiome.research/blob/main/Paper%20review/00181_Abstract.pdf)<br>
[ Keyword ] How UMAP works, Basic UMAP Parameters, Dertermining the number of clusters

## Manual


## Code

#### Build best quality control method on Metagenomic analysis
[Main : R code]()

* #### Practice
  * Feature selection with t.test & Evaluate with KNN algorithm [code](https://github.com/Jin-Baek/Human-microbiome.research/blob/main/practice/filtered_ttest.R) [/ data](https://github.com/Jin-Baek/Human-microbiome.research/blob/main/practice/2_practice_1.txt)
  * Discrete probability random variable & distribution [code](https://github.com/Jin-Baek/Human-microbiome.research/blob/main/practice/discreteRV.R)
  * Understand Correlation [code](https://github.com/Jin-Baek/Human-microbiome.research/blob/main/practice/Correlation.R) [/ data](https://github.com/Jin-Baek/Human-microbiome.research/blob/main/practice/2_practice_2.txt)
